### PR TITLE
v3 of the faq scraper

### PR DESCRIPTION
**Pull request by zappascout**
upgraded faq scraper, now executable with wrapper shell script + new name, outputting to file object with sys.stdout. also requirements file (just Requests). All seems to work :-)

### CODE REVIEW

File: scraper_faq_sheet.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          'link': 'https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html'
+        },
+    .
+    .
+    .
+    ]
+    etc
+In addition to an array of Question objects, each Section incorporates its title and a 'last updated' date value. For the Python intermediary stage (between CSV stream  and JSON) we will create a hierarchy of nested dictionaries and lists equivalent to the JSON objects and arrays.
+
+Execute with: $ python3 scraper_faq_sheet.py path/to/where/I/want/faq_json
```
Comment (Mr0grog) [inline scraper_faq_sheet.py:16]:
```suggestion
Execute with: $ python3 scraper_faq_sheet.py > path/to/where/I/want/faq.json
```

File: requirements-faq.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Requests==2.24.0
```
[the remark this answers is not in the document]
Comment (zappascout) [reply requirements-faq.txt:1]: Ooops, yes. We'd talked about needing a requirements file, but that was when the scraper script was in the other repo, which didn't have one - should have realized. It's gone now as far as I can see...

[the remark this answers is not in the document]
Comment (Mr0grog) [reply requirements-faq.txt:1]: It’s still shown here in the PR. Is it possible you didn’t commit the removal or maybe didn’t push the commit to GitHub?

[the remark this answers is not in the document]
Comment (zappascout) [reply requirements-faq.txt]: Doh! Yes, thought I'd commited but hadn't. Sorry for the confusion and thanks for your patience Rob...

**Pull request merged**